### ISSUE TITLE
Combobox examples: Mark up open button so it reflects combobox state

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#1333]
When using a touch based screen reader or using the reading cursor of a desktop screen reader, the open button of the combobox examples does not communicate the state of the combobox. Also the accessible "name of "Open" is insufficient.

Make the following changes so the open button is accessible:
1. Change the name to match the name of the element that the button controls. For example, in the comboboxes for choosing a "State", the name of the button should be "States".
2. Add aria-expanded to the button and set it to "true" when the combobox is expanded and "false when it is collapsed.
3. Add aria-controls to the button referencing the element that it controls.

**Comment by jongund** [comment]
@mcking65 

I added `aria-controls` and `aria-expanded` to the button, including updating the documentation and regression tests in [pull request 1336](https://github.com/w3c/aria-practices/pull/1336)
